pkg/kubelet/sysctl/whitelist.go

```diff
@@ -130,7 +121,7 @@ func (w *patternWhitelist) validateSysctl(sysctl string, hostNet, hostIPC bool)
 	return fmt.Errorf("%q not whitelisted", sysctl)
 }
 
-// Admit checks that all sysctls given in a v1.SysctlsPodAnnotationKey annotation
```

Comment:
Not only v1.SysctlsPodAnnotationKey, but also v1.UnsafeSysctlsPodAnnotationKey.

Reply:
s/in an annotationKey annotation/in annotations v1.SysctlsPodAnnotationKey and v1.UnsafeSysctlsPodAnnotationKey/?

Reply:
Done.